src/Microsoft.Health.Fhir.TemplateManagement/Utilities/DirectoryHelper.cs

```diff
@@ -0,0 +1,26 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.IO;
+using EnsureThat;
+
+namespace Microsoft.Health.Fhir.TemplateManagement.Utilities
+{
+    public static class DirectoryHelper
+    {
+        public static void ClearFolder(string directory)
+        {
+            EnsureArg.IsNotNullOrEmpty(directory, nameof(directory));
+
+            if (!Directory.Exists(directory))
+            {
+                return;
+            }
+
+            DirectoryInfo folder = new DirectoryInfo(directory);
+            folder.Delete(true);
```

Comment:
Will this delete the directory itself? If so, the function may better be renamed, as it not only cleans the files in the folder.

Reply:
Thanks, refined it.